### PR TITLE
make OOP work with preview 5 (bcb2)

### PR DESCRIPTION
**Pull request by heejaechang**
some version has changed in preview 5 branch. adding binding redirect and updating reference

### CODE REVIEW

**Comment by heejaechang** [comment]
test vsi please

**Comment by heejaechang** [comment]
@KevinH-MS @panopticoncentral @tannergooding can you take a look?

**Comment by heejaechang** [comment]
test vsi please

**Comment by heejaechang** [comment]
ping?

File: src/VisualStudio/Setup.Next/AssemblyRedirects.cs [hunk 1]
```diff
@@ -1,19 +1,15 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using Microsoft.VisualStudio;
 using Microsoft.VisualStudio.Shell;
 using Roslyn.VisualStudio.Setup;
 
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.EditorFeatures.Next.dll")]
 [assembly: ProvideRoslynBindingRedirection("Microsoft.VisualStudio.LanguageServices.Xaml.dll")]
+[assembly: ProvideRoslynBindingRedirection("Microsoft.VisualStudio.LanguageServices.Next.dll")]
```
Comment (KevinH-MS) [inline src/VisualStudio/Setup.Next/AssemblyRedirects.cs]: Do we already have multiple versions of this?

**Comment by KevinH-MS** [comment]
👍

**Pull request merged**